test/adaptive_load/plugin_util_test.cc

```diff
@@ -0,0 +1,378 @@
+#include "absl/status/status.h"
+#include "envoy/config/core/v3/base.pb.h"
+#include "envoy/registry/registry.h"
+
+#include "nighthawk/adaptive_load/input_variable_setter.h"
+#include "nighthawk/adaptive_load/metrics_plugin.h"
+#include "nighthawk/adaptive_load/scoring_function.h"
+#include "nighthawk/adaptive_load/step_controller.h"
+
+#include "external/envoy/source/common/config/utility.h"
+
+#include "api/adaptive_load/benchmark_result.pb.h"
+#include "api/adaptive_load/scoring_function_impl.pb.h"
+#include "api/client/options.pb.h"
+
+#include "adaptive_load/plugin_util.h"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace Nighthawk {
+
+namespace {
+
+const double kBadConfigThreshold = 98765.0;
```

Comment:
Considering the large scope for this constant - can we add a comment explaining what this is?

Reply:
Done